src/app/[locale]/components/organisms/events/CreateEvent.tsx

```diff
@@ -193,9 +193,16 @@ const CreateEvent = () => {
   const isMobile = useIsMobile()
   const handleSelectTimeZone = (zone: Zone) => {
     const result = addTimeZone(zone)
+
     const name = getCountry(zone.name)
     if (isMobile && result) {
-      toast(t('Toast.selectedTimezone', { name }), {
```

Comment:
Duration can be deleted when use specific type of toast (success, error). 

Check default duration on documentation:
react-hot-toast.com/docs/toast